.github/CONTRIBUTING.md

```diff
@@ -23,3 +25,43 @@ You can report bugs or make enhancement requests at the [Ansible GitHub issue pa
 Also please make sure you are testing on the latest released version of Ansible or the development branch; see the [Installation Guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) for details.
 
 Thanks!
+
+## DEVELOPER CERTIFICATE OF ORIGIN (DCO) VERSION 1.1
+```
```

Comment:
This will need to indicate the `text` type:

```suggestion
```text
```

Reply:
Also, it needs a blank line before it.

Reply:
Done - thanks for your patience with me ;)

Reply:
It looks like the blank line after the heading / before the text code block is missing.

Reply:
ugh, helps if you commit before you push.